Dockerfile.ci

```diff
@@ -278,7 +278,7 @@ ENV AIRFLOW_REPO=${AIRFLOW_REPO}\
 # * lazy-object-proxy<1.5.0: required by astroid
 # * pyjwt<2.0.0: flask-jwt-extended requires it
 # * dill<0.3.3 required by apache-beam
-ARG EAGER_UPGRADE_ADDITIONAL_REQUIREMENTS="lazy-object-proxy<1.5.0 pyjwt<2.0.0 dill<0.3.3 certifi<2021.0.0"
+ARG EAGER_UPGRADE_ADDITIONAL_REQUIREMENTS="lazy-object-proxy<1.5.0 pyjwt<2.0.0 dill<0.3.3 certifi<2021.0.0 botocore>=1.20.107,<1.21.0"
```

Comment:
Generally speaking here i think only upper bounds should be stated and only when this requirement is one of the transitive dependencies of 'airflow' package - so if you want to upper-bounds limit botocore in one of the  extras, AND setup.cfg install-requires already add botocore as transitive dependency, then you need to add the upper-bound limit here for botocore. You should never need to have lower-bound limit here.

Reply:
Hmm, okay I'll keep playing with it. This still wasn't enough to get it working and removing the lower limit made it even worse.

Btw, I think something is broken for these eager update PRs - it only tries to do the eager update when there is a single commit. If I add a second fixup commit, it installs with constraints and pip complains about the conflict. That's why I've been squashing my commits in this PR, but I haven't really dug into it yet.

Reply:
ah yeah might be ... worth aking a look after I come back from vacations

Reply:
Props to @uranusjr for figuring out what the conflict was (`amqp`).